### PR TITLE
TSE Review

### PR DESCRIPTION
**Pull request by carlosveigadev**
Added all the elements and style for the homepage, applying the bootstrap classes for the responsive design.

# Nav
- Defined the position
- Included Buttons and Logo.
- Included Dropdown button.

# Main
- Section and Articles
- Included the elements
- Added the paragraphs
- Embedded images
- Position the section and its elements
- Style the text and background color

# Footer
- Styled and positioned the footer lists
- Positioned all the list with its links

# Media query
- Added responsive design for 3 breakpoints (used bootstrap classes .sm .md and .lg )

Fixed the issues with recommended validations for Linter styling and W3 Validator

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: assets/css/media.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    font-size: 18px;
+    text-decoration: none;
+  }
+
+  #subs-btn,
+  #subs-btn-2 {
+    font-size: 15px;
+  }
+
+  #nav-menu {
```
Comment (andresporras3423) [inline assets/css/media.css]: Don't overuse ID selectors

File: assets/css/media.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    font-size: 26px;
+  }
+
+  .opinion {
+    border-bottom: 0 solid !important;
```
Comment (andresporras3423) [inline assets/css/media.css]: Don't use !important

File: assets/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,525 @@
...
+
+.heroes-txt h2 {
+  font-size: 15px;
+  color: #333;
+}
+
+.heroes-txt h2 a {
+  font-size: 15px;
+  color: #333;
+}
```
Comment (andresporras3423) [inline assets/css/styles.css]: Don't repeat yourself, in order to minimize code you can group selectors with the same style definition

File: assets/css/styles.css [hunk 4]
```diff
@@ -0,0 +1,525 @@
...
+  color: white;
+}
+
+#navbarDropdown {
+  color: red;
+  background-color: red;
+  border: 0 solid;
+}
+
+#navbarDropdown:hover {
```
Comment (andresporras3423) [inline assets/css/styles.css]: Don't use multiple ID selectors

File: assets/css/styles.css [hunk 5]
```diff
@@ -0,0 +1,525 @@
...
+  color: red;
+}
+
+.edit-pick div h2 a:hover {
+  text-decoration: underline red;
+}
+
+.feature div h2 a:hover {
+  text-decoration: underline red;
+}
```
Comment (andresporras3423) [inline assets/css/styles.css]: Don't repeat yourself, in order to minimize code you can group selectors with the same style definition

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @wrakc** 
Good job with your first submission for this project :muscle:  the description of your project is really clear and complete :clap: :clap: :clap:  Your page meets most of the requirements 
There are just a few details to improve before you can go ahead:
- [x] Don't use important
- [x] Don't overuse ID Selectors
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).
- [x]  Check the footer of your page, you have missed one of the links. Besides, check the border between the items of the list:



Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras
(for any question you can contact me in slack as @oscarporras02)

File: assets/css/media.css (hunk 2 above)
Comment (carlosveigadev) [reply assets/css/media.css]: Removed the !important from the .css files.

**Comment by carlosveigadev** [comment]
Thanks a lot @andresporras3423,

Here are the changes I've made to improve my project:
- Removed the !important from the .css files.
- Removed the use of ID's on the .css files.
- Applied the DRY concept to the .css files.
- Added the missed link and added the right borders to the large viewport.

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @wrakc ! 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good and your pull request looks extremely professional. 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [x] There is a horizontal scroll on the bottom of your project. Please see the screenshot below and fix this. 
Screen Shot 2020-06-16 at 12 49 07 AM

- [x] The articles in the "In the magazine" section are not quite aligned properly. Please compare your clone with the original page and fix this.
Your clone 👇 
Screen Shot 2020-06-16 at 12 49 33 AM
The original page 👇 
Screen Shot 2020-06-16 at 12 49 46 AM

### [OPTIONAL]

- [x] Please consider adding the `button-type` attribute to your button tags. You can find more information about button types in this [article](https://www.w3schools.com/tags/att_button_type.asp). 
* Screenshot from the `Webhint Report` section of your linter checks.

Screen Shot 2020-06-16 at 1 00 11 AM



****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by carlosveigadev** [comment]
Thanks a lot for the help @BrittanyBlake,

I made the improvements as asked:

- Removed the horizontal scroll from the page;
- Fixed the divs on the "In The Magazine" section;
- Added the type: button to the buttons.

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Status: Approved ✔

## Awesome project @wrakc 👏🙌!! It's a 👍 from me.

### Consider making these changes before merging.

### [OPTIONAL]

- The whole webpage except the `subscribe/newsletter` and footer, should have some space on either side.
- The text below the magazine cover in `subscribe/newsletter` section is encroaching on the image. Even though this is the case for the original site, you can improve on it 😉.

image

Onto the next project 🚀

Happy coding 👨‍💻

**Comment by carlosveigadev** [comment]
Thanks a lot @tirthajyoti-ghosh!

I added the margins as requested.

**Pull request merged**